option/scheduled_lambda.go

```diff
@@ -0,0 +1,50 @@
+package option
+
+// ScheduledLambdaOptions is used to hold optional scheduled lambda settings
+type ScheduledLambdaOptions struct {
+	Tag          *string
+	TotalTimesToExecute *int64
+	Version *string
+	WebhookAuthHeader *string
+	WebhookPayload *string
+	WebhookUrl *string
+	ResumePermanentError *bool
+}
```

Comment:
I think we should start to just embed the request struct instead, as that way we at least get some stuff for free when a new field is added
```suggestion
type ScheduledLambdaOptions struct {
	openapi.CreateScheduledLambdaRequest
}
```

and then also add a `option.WithScheduledLambdaRequest(r openapi.CreateScheduledLambdaRequest)` which can be used when there is no specific helper